lightning/src/blinded_path/message.rs

```diff
@@ -37,7 +37,7 @@ use core::ops::Deref;
 /// A blinded path to be used for sending or receiving a message, hiding the identity of the
 /// recipient.
 #[derive(Clone, Debug, Hash, PartialEq, Eq)]
-pub struct BlindedMessagePath(pub(super) BlindedPath);
+pub struct BlindedMessagePath(pub(crate) BlindedPath);
```

Comment:
Shouldn't need to change this anymore.

Reply:
Done! [pr3177.18](https://github.com/shaavan/rust-lightning/commits/pr3177.18)
Thanks for the pointer!